src/main/java/org/cryptomator/ui/mainwindow/VaultListController.java

```diff
@@ -127,6 +144,39 @@ public void initialize() {
 		root.setOnDragOver(this::handleDragEvent);
 		root.setOnDragDropped(this::handleDragEvent);
 		root.setOnDragExited(this::handleDragEvent);
+
+		initAddVaultBtnContextMenu();
+	}
+
+	private void initAddVaultBtnContextMenu() {
+		ContextMenu contextMenu = new ContextMenu();
+		FontAwesome5IconView addIcon = new FontAwesome5IconView();
+		addIcon.setGlyph(FontAwesome5Icon.PLUS);
+		addIcon.setTextAlignment(TextAlignment.CENTER);
+		addIcon.setWrappingWidth(14);
+
+		MenuItem item1 = new MenuItem(resourceBundle.getString("addvaultwizard.welcome.newButton"), addIcon);
```

Comment:
Bad naming. How about "createNewMenuItem"?

Reply:
renamed to "newVaultMenuItem"